saleor/webhook/tests/test_webhook_serializers.py

```diff
@@ -73,52 +75,112 @@ def test_serialize_product_attributes(
     ]
 
 
-@pytest.mark.parametrize(
-    "taxes_included, taxes_calculated",
-    [
-        (True, True),
-        (True, False),
-        (False, True),
-        (False, False),
-    ],
+ATTRIBUTES = sentinel.ATTRIBUTES
+
+
+@patch(
+    "saleor.webhook.serializers.serialize_product_or_variant_attributes",
+    new=Mock(return_value=ATTRIBUTES),
 )
-def test_serialize_checkout_lines(
-    checkout_with_items_for_cc, taxes_included, taxes_calculated, site_settings
+def test_serialize_checkout_lines_with_taxes(
+    checkout_with_prices,
+    mocked_fetch_checkout,
 ):
     # given
+    checkout = checkout_with_prices
+    lines, _ = fetch_checkout_lines(checkout)
+    manager = get_plugins_manager()
+    checkout_info = fetch_checkout_info(checkout, lines, [], manager)
+
+    # when
+    checkout_lines_data = serialize_checkout_lines_with_taxes(
+        checkout_info, manager, lines, []
+    )
+
+    # then
+
+    data_len = 0
+    for data, line_info in zip(checkout_lines_data, lines):
+        line = line_info.line
+        variant = line.variant
+        product = variant.product
+        collections = list(product.collections.all())
+        variant_channel_listing = None
+
+        for channel_listing in line.variant.channel_listings.all():
+            if channel_listing.channel_id == checkout.channel_id:
+                variant_channel_listing = channel_listing
+
+        if not variant_channel_listing:
+            continue
+
+        base_price = variant.get_price(
+            product, collections, checkout.channel, variant_channel_listing
+        )
+        currency = checkout.currency
+        assert data == {
+            "id": graphene.Node.to_global_id("CheckoutLine", line.pk),
+            "sku": variant.sku,
+            "quantity": line.quantity,
+            "charge_taxes": product.charge_taxes,
+            "base_price": quantize_price(base_price.amount, currency),
+            "price_net_amount": ANY,
```

Comment:
Can we change it to real values? It is important for us, to send values after quantitze them, so would be good to check this. (DB  stores values with 3 number after coma: Decimal(10.000), webhook payload should send Decimal(10.00)

Reply:
This comment is related to all tests related to price fields, where we used `ANY`